### PR TITLE
Update webrtc doc

### PR DESCRIPTION
**Pull request by sangchul1011**
- ACR: ACR-1702, ACR-1712

### CODE REVIEW

File: docs/application/native/guides/multimedia/webrtc.md [hunk 1]
```diff
@@ -60,16 +60,59 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
+
+    ret = webrtc_create(&webrtc);
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_MIC, &a_src_id);
+    ret = webrtc_media_source_set_transceiver_codec(webrtc, a_src_id, WEBRTC_MEDIA_TYPE_AUDIO, WEBRTC_TRANSCEIVER_CODEC_PCMU);
+    ...
+    ret = webrtc_start(webrtc);
+    ret = webrtc_create_offer(webrtc, NULL, &offer_desc);
+    ```
+
+    In the following example code, it creates offer description which includes an audio stream with OPUS codec of `WEBRTC_TRANSCEIVER_DIRECTION_SENDRECV` and a video stream with H264 codec of `WEBRTC_TRANSCEIVER_DIRECTION_RECVONLY`:
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:83]:
```suggestion
    In the following example code, it creates an offer description which includes an audio stream with OPUS codec of `WEBRTC_TRANSCEIVER_DIRECTION_SENDRECV` and a video stream with H264 codec of `WEBRTC_TRANSCEIVER_DIRECTION_RECVONLY`:
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 2]
```diff
@@ -78,9 +121,29 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
     > [!NOTE]
     >
     > Muting a media source means it sends black video frames or silent audio frames to a remote peer. Mute or un-mute of a media source is also possible in `WEBRTC_STATE_PLAYING`.
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:123]:
```suggestion
    > Muting a media source means it sends black video frames or silent audio frames to a remote peer. Mute or unmute of a media source is also possible in `WEBRTC_STATE_PLAYING`.
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 3]
```diff
@@ -78,9 +121,29 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
     > [!NOTE]
     >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:122]:
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 4]
```diff
@@ -60,16 +60,59 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
+    >
+    > It is recommended to use `WEBRTC_MEDIA_SOURCE_TYPE_NULL` when you need to set `WEBRTC_TRANSCEIVER_DIRECTION_RECVONLY` of a transceiver which currently belongs to a media source. If so, the media source of this type could give you more codec selection because it does not require to consider encoder situation on the local target device.
+
+4. To pause a media source, use the `webrtc_media_source_set_pause()`:
 
     ```c
     ret = webrtc_media_source_set_pause(webrtc, a_src_id, WEBRTC_MEDIA_TYPE_AUDIO, true);
     ```
     > [!NOTE]
     >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:111]:
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 5]
```diff
@@ -60,16 +60,59 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_MIC, &a_src_id); /* Default direction is WEBRTC_TRANSCEIVER_DIRECTION_SENDRECV */
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_NULL, &v_src_id); /* Default direction is WEBRTC_TRANSCEIVER_DIRECTION_RECVONLY */
+    ret = webrtc_media_source_set_transceiver_codec(webrtc, a_src_id, WEBRTC_MEDIA_TYPE_AUDIO, WEBRTC_TRANSCEIVER_CODEC_OPUS);
+    ret = webrtc_media_source_set_transceiver_codec(webrtc, v_src_id, WEBRTC_MEDIA_TYPE_VIDEO, WEBRTC_TRANSCEIVER_CODEC_H264);
+    ...
+    ret = webrtc_start(webrtc);
+    ret = webrtc_create_offer(webrtc, NULL, &offer_desc);
+    ```
+    > [!NOTE]
+    >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:102]: There should be no empty line between [!NOTE] and the following text.
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 6]
```diff
@@ -78,9 +121,29 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
+    ...
+    ret = webrtc_start(webrtc);
+    ...
+    ret = webrtc_media_source_get_encoder_bitrate(webrtc, v_src_id, WEBRTC_MEDIA_TYPE_VIDEO, &target_bitrate);
+    /* some logic to increase or decrease the current target bitrate */
+    /* e.g. new_bitrate = target_bitrate / 2; */
+    ret = webrtc_media_source_set_encoder_bitrate(webrtc, v_src_id, WEBRTC_MEDIA_TYPE_VIDEO, new_bitrate);
+    ```
+    > [!NOTE]
+    >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:143]:
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 7]
```diff
@@ -89,7 +152,7 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
     >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:152]:
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 8]
```diff
@@ -98,6 +161,24 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:161]:
```suggestion
```

File: docs/application/native/guides/multimedia/webrtc.md [hunk 9]
```diff
@@ -98,6 +161,24 @@ You can add media sources to a webrtc handle. Once you get source id of the medi
...
+
+    ret = webrtc_create(&webrtc);
+    ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_CAMERA, &cam_src_id);
+    /* To try to use a particular camera device id */
+    ret = webrtc_camera_source_set_device_id(webrtc, cam_src_id, 1);
+    ...
+    ret = webrtc_start(webrtc);
+    ```
+    > [!NOTE]
+    >
```
Comment (safir-srbd) [inline docs/application/native/guides/multimedia/webrtc.md:179]:
```suggestion
```

**Review by safir-srbd — COMMENTED** [review]
@sangchul1011 Hello. Hope you are well. I have gone through the PR, and made some suggestions below. Please implement the changes.

File: docs/application/native/guides/multimedia/webrtc.md (hunk 1 above)
Comment (sangchul1011) [reply docs/application/native/guides/multimedia/webrtc.md]: done, thanks.

File: docs/application/native/guides/multimedia/webrtc.md (hunk 2 above)
Comment (sangchul1011) [reply docs/application/native/guides/multimedia/webrtc.md]: done, thanks.

File: docs/application/native/guides/multimedia/webrtc.md (hunk 3 above)
Comment (sangchul1011) [reply docs/application/native/guides/multimedia/webrtc.md]: done, thanks.

**Review by safir-srbd — APPROVED** [review]
@sangchul1011 Thank you for making the suggested changes.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**